### PR TITLE
enh(epp): applications-monitoring-centreon-sql-metrics

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [X] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-sql-metrics.md [hunk 1]
```diff
@@ -5,39 +5,120 @@ title: Centreon SQL Metrics
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-sql-metrics.md:7]: old text with link to the watch?
or clearer explanations as to what the connector actually does?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-sql-metrics.md [hunk 2]
```diff
@@ -5,39 +5,120 @@ title: Centreon SQL Metrics
...
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="App-Monitoring-Centreon-SQL-Metrics-custom" label="App-Monitoring-Centreon-SQL-Metrics-custom">
 
-Le connecteur de supervision Centreon SQL Metrics apporte un modèle d'hôte :
+| Alias                | Modèle de service                                       | Description                                                                              |
+|:---------------------|:--------------------------------------------------------|:-----------------------------------------------------------------------------------------|
+| Notifications-Count  | App-Monitoring-Centreon-SQL-Notifications-Count-custom  | Contrôle le nombre de nouvelles notifications                                            |
+| Poller-Delay         | App-Monitoring-Centreon-SQL-Poller-Delay-custom         | Contrôle le temps de la dernière communication entre le collecteur et le serveur central |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-sql-metrics.md:25]:
```suggestion
| Poller-Delay         | App-Monitoring-Centreon-SQL-Poller-Delay-custom         | Contrôle le temps écoulé depuis la dernière communication entre le collecteur et le serveur central |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-sql-metrics.md [hunk 3]
```diff
@@ -5,39 +5,120 @@ title: Centreon SQL Metrics
...
+<TabItem value="Storage-Partitioning" label="Storage-Partitioning">
+
+Pas de métriques.
 
 </TabItem>
-<TabItem value="Virtual-Curve" label="Virtual-Curve">
+<TabItem value="Virtual-Service" label="Virtual-Service">
 
-Les métriques dépendent de la configuration du service. Voir l'article sur [The Watch](https://thewatch.centreon.com/product-how-to-21/get-to-know-app-centreon-sql-metric-pack-and-start-building-some-virtual-curves-296).
+Les métriques sont personnalisables.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-sql-metrics.md:121]: text?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-sql-metrics.md [hunk 4]
```diff
@@ -49,60 +130,228 @@ Le collecteur exécutant le contrôle doit pouvoir se connecter au serveur de ba
 
 L'utilisateur doit avoir les droits de réaliser un 'SELECT' sur les tables **index_data**, **metrics** et **instances** de la base **centreon_storage**.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-sql-metrics.md:131]:
```suggestion
L'utilisateur doit avoir le droit de réaliser un 'SELECT' sur les tables **index_data**, **metrics** et **instances** de la base **centreon_storage**. 
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/applications-monitoring-centreon-sql-metrics.md [hunk 5]
```diff
@@ -5,39 +5,120 @@ title: Centreon SQL Metrics
...
+
+</TabItem>
+<TabItem value="Storage-Partitioning" label="Storage-Partitioning">
+
+No metrics.
+
+</TabItem>
+<TabItem value="Virtual-Service" label="Virtual-Service">
+
+Custom metrics. It depends of the service configuration.
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-monitoring-centreon-sql-metrics.md]:
```suggestion
Metrics depend on the service configuration. Check our [The Watch blog post](https://thewatch.centreon.com/product-how-to-21/get-to-know-app-centreon-sql-metric-pack-and-start-building-some-virtual-curves-296).
```

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**